pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
That's the allocators job

On Oct 7, 2015, at 11:05 PM, Paul Morie <notifications@github.com> wrote:

In pkg/security/util.go
<https://github.com/openshift/origin/pull/4996#discussion_r41472147>:

> @@ -2,6 +2,7 @@ package security
>
>  const (
>  	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
> +	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"

I think we need to differentiate FSGroup from the allowed supplemental
groups. There are use-cases where the space of reserved GIDs for FSGroup is
going to be separate from the space used for shared storage.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4996/files#r41472147>.